trulens_eval/trulens_eval/schema.py

```diff
@@ -439,6 +442,7 @@ def __init__(
         kwargs['app_id'] = "temporary"  # will be adjusted below
         kwargs['feedback_mode'] = feedback_mode
         kwargs['tags'] = ""
```

Comment:
if we want tags and metadata to impact the generated app_id, set them here

Reply:
Not clear if it should or should not denote a new app_id. Will re-evaluate as usage comes in